noisemaker/effects.py

```diff
@@ -250,10 +250,7 @@ def post_process(tensor, shape, freq, ridges_hint=False, spline_order=3, reflect
     if with_convolve:
         for kernel in with_convolve:
             if isinstance(kernel, str):
-                if kernel not in ValueMask:
-                    kernel = 'conv2d_{}'.format(kernel)
-
-                kernel = ValueMask[kernel]
+                kernel = ValueMask['conv2d_{}'.format(kernel)]
```

Comment:
I guess this wasn't the original intent, but I think it's what it actually did...